### DISCUSSION TITLE
G DATA virus scanner recognizes the CefSharp.dll as a Trojan

### DISCUSSION THREAD

**Discussion by kelemenattila** [cefsharp/CefSharp#4863] [General]
Hi, 
I updated the CefSharp to version 125.0.210.0
We have a problem with CefSharp dll. G DATA virus scanner recognizes as a Trojan Virus
![image](https://github.com/cefsharp/CefSharp/assets/48238058/36efefe6-db66-4c74-be9b-357c3322544d)

I'm using BitDefender. With this antivirus no problem

**Comment by kelemenattila** [comment]
I made a full scan on VirusTotal and 6 from 74 Virus Scanner recognizes as Trojan :( 
![image](https://github.com/cefsharp/CefSharp/assets/48238058/008ed3b1-5eea-4e73-89dd-e6a0d1be8879)

**Comment by amaitland** [comment]
I'm open to suggestions on actionable steps here. Anyone is welcome to review the code.

My guess would be that it's not the code itself that's the problem, it's that others have used `CefSharp` too write malware in the past. There are hazards of being an open source project, anyone can use the project.
